add-seed-config.sh

```diff
@@ -0,0 +1,5 @@
+SEED_PEER_ID=`LNK_HOME=/tmp/seed-home ./lnk.sh profile peer`
+PROFILE_ID_1=`cat /tmp/link-local-1/active_profile`
```

Comment:
Note that another way to do this would be `LNK_HOME=/tmp/link-local-1 lnk profile peer`. I think this `cat ...` approach is fine here though.

Reply:
I just found that actually the active profile id is different from peer id: 

```
$ cat /tmp/link-local-1/active_profile 
d70bb735-3b11-4464-84fe-cb77cf40f8da

$ LNK_HOME=/tmp/link-local-1 lnk profile peer
hybtzdmirsdsaociewmqhs6ejzzm7r8qj3jy6hbz165kbkincfrsn6
```

Reply:
or you probably meant `LNK_HOME=/tmp/link-local-1 lnk profile get` , that will be the same as the active_profile.